nengo/spa/vocab.py

```diff
@@ -379,6 +400,33 @@ def prob_cleanup(self, similarity, vocab_size, steps=10000):
         pcorrect = (1 - perror1) ** vocab_size
         return pcorrect
 
+    def extend(self, keys, unitary=False):
+        """Extends the vocabulary with additional keys.
+
+        Creates and adds the semantic pointers listed in keys to the
+        vocabulary.
+
+        Parameters
+        ----------
+        keys : list of strings
+            List of semantic pointer names to be added to the vocabulary.
+
+        """
+        if is_iterable(unitary):
```

Comment:
I think, that should be documented because it is not obvious that `unitary` excepts a list of strings.